src/runner.js

```diff
@@ -4,13 +4,24 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {object} [options.formater] Changing default formater used (MessageFormat)
+   * @param {string} [options.language] Options for the language of the formater
+   * @memberof Runner
+   */
+  constructor(options) {
     this.yarnNodes = {};
     this.variables = new DefaultVariableStorage();
     this.functions = {};
     this.visited = {}; // Which nodes have been visited
+    this.options = options || {};
+    this.options.language = options.language || 'en';
```

Comment:
`should be this.options.language = this.options.language || 'en';`

to protect against options being undefined

Reply:
I used default values on constructor before but It fail pass tests because it was included in node later on

If we switch back to old version compat we can't uses that